engine/src/test/java/io/zeebe/engine/util/TestStreams.java

```diff
@@ -444,7 +444,11 @@ public void close() throws Exception {
         return;
       }
 
-      asyncSnapshotDirector.closeAsync().join();
+      try {
+        asyncSnapshotDirector.closeAsync().join();
+      } catch (final Exception e) {
+        Loggers.IO_LOGGER.debug("Close snapshot director failed", e);
```

Comment:
Why this failed?

Reply:
When we try to close asyncSnapshotDirector, Stream processor actor can be already closed because it has failed. AsynsSnapshotDirector tries to get the lastCommitPosition but fails because StreamProcessor is closed/failed.

Reply:
this is no longer true due to removing the enforcing snapshot